Documentation/contributing/release/backports.rst

```diff
@@ -65,8 +65,15 @@ One-time Setup
       $ git config --global user.name "John Doe"
       $ git config --global user.email johndoe@example.com
 
+#. Add remotes for the Cilium upstream repository and your Cilium repository fork.
+
+   .. code-block:: bash
```

Comment:
Nit:
```suggestion
   .. code-block:: shell-session
```

Reply:
I didn't use `shell-session` here because there is no command output to display here.

Reply:
It doesn't have to. Using `shell-session` would colour the `$` prompt differently, and would prevent it to be selected with the rest of the commands on text selection. But that's very minor, and I see the block above already uses `bash` too. Let's keep it this way.